packages/discount-code-generator/src/cli.js

```diff
@@ -144,13 +144,20 @@ const resolveInput = (_args) => {
         .on('data', (data) => {
           const arrayDelim = _args.multiValueDelimiter
           // Add condition so module doesn't fail if there are no cartDiscounts
-          if (data.cartDiscounts)
-          // eslint-disable-next-line no-param-reassign
+          if (data.cartDiscounts) {
+            // eslint-disable-next-line no-param-reassign
             data.cartDiscounts = data.cartDiscounts.split(arrayDelim)
-          _attributes += data
+            // eslint-disable-next-line no-param-reassign
+            data.cartDiscounts = data.cartDiscounts.map(cartDiscount => ({
+              typeId: 'cart-discount',
+              id: cartDiscount,
+            }),
+            )
+          }
+          _attributes += JSON.stringify(data)
```

Comment:
@wizzy25  please can you clarify how this should work? `_attributes` is a string which can contain multiple JSON arrays? Shouldn't there be also some delimiter or maybe it should be an array instead of a string?

Because at the end you are calling `unflatten(JSON.parse(_attributes))` which will throw an error if there are concatenated multiple JSON arrays without any delimiter

What about to do this?
```js
const _attributes = []
_attributes.push(data)
_attributes.push(data)
unflatten(_attributes)